pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -1259,6 +1263,53 @@ func (l *secretLoader) AddToGraph(g osgraph.Graph) error {
 	return nil
 }
 
+type pvcLoader struct {
+	namespace string
+	lister    kclient.PersistentVolumeClaimsNamespacer
+	items     []kapi.PersistentVolumeClaim
+}
+
+func (l *pvcLoader) Load() error {
+	list, err := l.lister.PersistentVolumeClaims(l.namespace).List(kapi.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	l.items = list.Items
+	return nil
+}
+
+func (l *pvcLoader) AddToGraph(g osgraph.Graph) error {
+	for i := range l.items {
+		kubegraph.EnsurePersistentVolumeClaimNode(g, &l.items[i])
+	}
+
+	return nil
+}
+
+type nodeLoader struct {
```

Comment:
We can't have this.  Its a problem similar to the `HostPort` problem.  Project admins and editors don't have the rights to see nodes.  They know that some exist, so we can do things like "is your service spread across multiple hosts", but we can't do a get on this resource type for status.